doc/release.md

```diff
@@ -4,27 +4,13 @@ So, you want to release the `X.Y.Z` version of astroid ?
 
 ## Process
 
-1. Preparation
-   1. Check if the dependencies of the package are correct
-   2. Put the version numbers, and the release date into the changelog
-   3. Generate the new copyright notices for this release:
+1. Check if the dependencies of the package are correct
+2. Install the release dependencies `pip3 install pre-commit tbump`
+3. Bump the version and release by using `tbump X.Y.Z --no-push`.
+4. Check the result.
 
-```bash
-pip3 install copyrite
-copyrite --contribution-threshold 1 --change-threshold 3 --backend-type \
-git --aliases=.copyrite_aliases . --jobs=8
-# During the commit pre-commit and pyupgrade will remove the encode utf8
-# automatically
-```
-
-4. Submit your changes in a merge request and make sure the tests are passing on
-   Travis/GithubActions: https://travis-ci.org/PyCQA/astroid/
-
-5. Do the actual release by tagging the master with `vX.Y.Z` (ie `v1.6.12` or `v3.0.0a0`
-   for example).
-
-Until the release is done via Travis or GitHub actions on tag, run the following
-commands:
+5. Until the release is done via Travis or GitHub actions on tag, run the following
+   commands:
```

Comment:
If you like, take a look at this script here: [cdce8p/python-typing-update -> release.yaml]( https://github.com/cdce8p/python-typing-update/blob/f79966eb104f220fdcc78f87764eb74ab6e527c2/.github/workflows/release.yaml). It's the release workflow for one of my personal projects. That should also work for pylint / astroid. The only requirement is that `PYPI_API_TOKEN` is added as secret.

Reply:
I'm going to try to release a dev version in order to test that. I don't want to be surprised when releasing ``2.6.0`` 😄